demos/projects/ST/stm32h745i-disco/cm7/config/demo_config.h

```diff
@@ -487,4 +487,16 @@ static unsigned char root_cert_array[] = {
  */
 #define democonfigIOTHUB_PORT          ( 8883 )
 
+#ifdef democonfigENABLE_DPS_SAMPLE
+    #define democonfigCHUNK_DOWNLOAD_SIZE    256
```

Comment:
might be able to bump these

Reply:
Will align like in the others but this is going to stay at 1k for now since there seems to be a receive buffer with LWIP that is small. I haven't drilled into that yet but it (currently) doesn't affect performance in a huge way since its over ethernet.